src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/TestingControllers/ApplicationsControllerTests.cs

```diff
@@ -461,22 +457,103 @@ public async void GetAll_ReturnsOK()
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         }
 
-        private static DocumentClientException CreateDocumentClientExceptionForTesting(string message, HttpStatusCode httpStatusCode)
+        /// <summary>
+        /// Request all applications of an invalid app owner. Repository called is never called. Controller returns bad request.
+        /// </summary>
+        [Fact]
+        public async void GetMany_InvalidOrg_ReturnsBadRequest()
+        {
+            // Arrange
+            string requestUri = $"{BasePath}/applications/test%20org";
+
+            Mock<IApplicationRepository> applicationRepository = new Mock<IApplicationRepository>();
+
+            HttpClient client = GetTestClient(applicationRepository.Object);
+
+            // Act
+            HttpResponseMessage response = await client.GetAsync(requestUri);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
```

Comment:
In cases like this, where org is part of the url, it might be better to return NotFound. I assume BadRequest was already in use?

Reply:
Looked at the code. Old stuff. Leave as is for now.